internal/charmstore/search.go

```diff
@@ -580,6 +598,9 @@ func nameFilter(value string) elasticsearch.Filter {
 // ownerFilter generates a filter that will match against the
 // owner taken from the URL.
 func ownerFilter(value string) elasticsearch.Filter {
```

Comment:
We should probably add another filter in search query, ie: promulgated=1 - not removing this one for compatibility. "owner=" semantics is somehow "broken" with new way of looking at promulgated charms.